### PR TITLE
Moved 'Query' folder from Witsml project to WitsmlExplorer.Api

### LINKED ISSUE #471: 👉 Refactor query code for Wells
**👉 Is your feature request related to a problem? Please describe.**
Related to #350 
Sub-task for #350 

**Describe the solution you'd like**
In this task, we do refactor for WELL object type to maintain clarity in the changes and to allow proper testing

### PR DESCRIPTION
**Pull request by kdi-chandra**
# Request Template Witsml Explorer
Thank you for contributing to WITSML Explorer!
Before submitting this PR, please fill in the following template.

## Fixes
This pull request fixes #471, #350 

## Description
1. Query folder is moved from 'Witsml' project to 'WitsmlExplorer.Api'
2. Refactored 'Wells' related query code

## Type of change
_List the types of change. Remove from the list any points which are not necessary_

* Enhancement of existing functionality/code quality
* Refactor 

## Impacted Areas in Application
_List general components of the application that this PR will affect:_

WitsmlExplorer.API

## Checklist:
_Please tick all the boxes or remove the ones that aren't needed and explain why_

*Communication*
* [x] PR is related to an issue
* [ ] I have made corresponding changes to the documentation

*Code quality*
* [x] Code follows the style guidelines
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] Existing tests pass
* [x] New code is covered by passing tests

## Further comments
_If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc..._

### CODE REVIEW

File: Src/WitsmlExplorer.Api/Services/WellService.cs [hunk 1]
```diff
@@ -40,8 +41,8 @@ public async Task<IEnumerable<Well>> GetWells()
         private async Task<IEnumerable<Well>> GetWellsInformation(string wellUid = null)
         {
             var start = DateTime.Now;
-            var query = string.IsNullOrEmpty(wellUid) ? WellQueries.QueryAll() : WellQueries.QueryByUid(wellUid);
-            var result = await WitsmlClient.GetFromStoreAsync(query, OptionsIn.Requested);
+            WitsmlWells witsmlWells = string.IsNullOrEmpty(wellUid) ? WellQueries.GetAllWitsmlWells() : WellQueries.GetWitsmlWellByUid(wellUid);
```
Comment (lindsve) [inline Src/WitsmlExplorer.Api/Services/WellService.cs]: use `var`

File: Src/WitsmlExplorer.Api/Workers/CreateWellWorker.cs [hunk 2]
```diff
@@ -33,7 +33,7 @@ public async Task<(WorkerResult, RefreshAction)> Execute(CreateWellJob job)
             var well = job.Well;
             Verify(well);
 
-            var wellToCreate = CreateRequest(well);
+            WitsmlWells wellToCreate = WellQueries.CreateWitsmlWell(well);
```
Comment (lindsve) [inline Src/WitsmlExplorer.Api/Workers/CreateWellWorker.cs]: use `var`

File: Src/WitsmlExplorer.Api/Workers/DeleteWellWorker.cs [hunk 3]
```diff
@@ -28,8 +29,8 @@ public async Task<(WorkerResult, RefreshWell)> Execute(DeleteWellJob job)
         {
             var wellUid = job.WellReference.WellUid;
 
-            var query = WellQueries.DeleteQuery(wellUid);
-            var result = await witsmlClient.DeleteFromStoreAsync(query);
+            WitsmlWells witsmlWell = WellQueries.DeleteWitsmlWell(wellUid);
```
Comment (lindsve) [inline Src/WitsmlExplorer.Api/Workers/DeleteWellWorker.cs]: `var`

File: Src/WitsmlExplorer.Api/Workers/ModifyWellWorker.cs [hunk 4]
```diff
@@ -31,9 +32,9 @@ public async Task<(WorkerResult, RefreshAction)> Execute(ModifyWellJob job)
             Verify(job.Well);
             var wellUid = job.Well.Uid;
             var wellName = job.Well.Name;
-            var query = CreateUpdateQuery(job.Well);
+            WitsmlWells witsmlWellToUpdate = WellQueries.UpdateWitsmlWell(job.Well);
```
Comment (lindsve) [inline Src/WitsmlExplorer.Api/Workers/ModifyWellWorker.cs]: `var`

**Review by lindsve — COMMENTED** [review]
Some nitpicking about using `var`, otherwise 👍  from me

File: Src/WitsmlExplorer.Api/Workers/DeleteWellWorker.cs (hunk 3 above)
Comment (kdi-chandra) [reply Src/WitsmlExplorer.Api/Workers/DeleteWellWorker.cs]: updated

**Review by lindsve — APPROVED** [review]

**Review by HakonHesselberg — APPROVED** [review]
*Reviewer checklist*
- [x] Read issue and PR
- [ ] Pulled branch and manually tested
- [ ] Verify that PR resolves issue
- [x] Reviewed the code

**Review by steinsiv — APPROVED** [review]
Looks good 👍 
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue
- [x] Reviewed the code

**Pull request merged**